src/script/components/FadingScrollbar.tsx

```diff
@@ -0,0 +1,128 @@
+/*
+ * Wire
+ * Copyright (C) 2021 Wire Swiss GmbH
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see http://www.gnu.org/licenses/.
+ *
+ */
+
+import React, {forwardRef, useEffect, useState} from 'react';
+
+import {debounce} from 'underscore';
+
+const config = {
+  ANIMATION_SPEED: 0.05,
+  DEBOUNCE_THRESHOLD: 1000,
+};
+
+function parseColor(color: string) {
+  const el = document.body.appendChild(document.createElement('thiselementdoesnotexist'));
+  el.style.color = color;
+  const col = getComputedStyle(el).color;
+  document.body.removeChild(el);
+  const [, r, g, b, a = 1] = /rgba?\((\d+), *(\d+), *(\d+),? *(\d*\.?\d*)?\)/.exec(col) ?? [0, 0, 0, 0, 1];
+  return [+r, +g, +b, +a];
+}
+
+const useFadingScrollbar = () => {
```

Comment:
The code of this function has not changed. Not sure why git thinks it's a new file, should have been detected as a moved file